tests/crashes/133066.rs

```diff
@@ -0,0 +1,12 @@
+//@ known-bug: #133066
```

Comment:
Question: can this be further minimized?

Reply:
I spent a couple of minutes on this and didn't manage to produce any meaningful difference :shrug: